jf_agent/diagnostics.py

```diff
@@ -26,7 +26,9 @@ def _write_diagnostic(obj):
 
 def capture_agent_version():
     git_head_hash = os.getenv('SHA')
+    build_timestamp = os.getenv('BUILDTIME')
     _write_diagnostic({'type': 'agent_version', 'sha': git_head_hash})
+    _write_diagnostic({'type': 'agent_version', 'timestamp': build_timestamp})
```

Comment:
It's probably a bit more consistent with the other diagnostics to include these into the same item, so instead do:
```
_write_diagnostic({'type': 'agent_version', 'sha': git_head_hash, 'timestamp': build_timestamp})
```